lib/tasks/check_sessions.rake

```diff
@@ -0,0 +1,37 @@
+desc 'check for stale sessions'
```

Comment:
the algorithm here is very naive, since it assumes (by default) a 3-minute window where no idle window pings have hit the server. Seemed best to start with something and then refine.